package.json

```diff
@@ -62,7 +62,6 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-intl": "^5.8.0",
-    "react-redux": "^7.2.8",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "redux": "^4.2.0",
```

Comment:
Should redux and redux-observable also be removed?

Reply:
@Terala-Priyanka , you may be right that other dev-deps/peer-deps related to redux can be removed. This work was narrowly scoped to `react-redux`, but please feel free to test whether other deps can be removed and clean up some cruft!